actors/common/src/main/java/org/sunbird/learner/actors/OnDemandSchedulerActor.java

```diff
@@ -0,0 +1,78 @@
+package org.sunbird.learner.actors;
+
+import org.sunbird.actor.core.BaseActor;
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.*;
+import org.sunbird.common.quartz.scheduler.OnDemandSchedulerManager;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.sunbird.common.request.orgvalidator.BaseOrgRequestValidator.ERROR_CODE;
+
+/** @author Amit Kumar */
+@ActorConfig(
+  tasks = {"onDemandStartScheduler"},
+  asyncTasks = {}
+)
+public class OnDemandSchedulerActor extends BaseActor {
+  static final String TYPE = "type";
+
+  @Override
+  public void onReceive(Request actorMessage) throws Throwable {
+    if (actorMessage
+            .getOperation()
+            .equalsIgnoreCase(ActorOperations.ONDEMAND_START_SCHEDULER.getValue())) {
+     startSchedular(actorMessage);
+    } else {
+      ProjectLogger.log("UNSUPPORTED OPERATION");
```

Comment:
I think we can also give Log-Level as error here

Reply:
ok. done.